plugins/html.js

```diff
@@ -0,0 +1,68 @@
+const html2json = require("html2json").html2json;
+
+module.exports = function (multimeter) {
+    multimeter.console.on('addLines', function (event) {
+        if (event.type === 'log') {
+            event.line = parseLogJson(html2json(event.line));
+            event.formatted = true;
+        }
+    });
+};
+
+let parseLogJson = function (obj) {
+    let ret = "",
+        bgColor,
+        color,
+        bold,
+        underline;
+
+    if (obj.attr && obj.attr.style) {
+        let i = obj.attr.style.indexOf("color:");
+        if (i !== -1) {
+            color = `${obj.attr.style[i + 1].replace(/;/g, "")}-fg`;
+        }
+
+        i = obj.attr.style.indexOf("background:");
+        if (i !== -1) {
+            bgColor = `${obj.attr.style[i + 1].replace(/;/g, "")}-bg`;
+        }
+
+        i = obj.attr.style.indexOf("font-weight:");
+        if (i !== -1) {
+            bold = `${obj.attr.style[i + 1].replace(/;/g, "")}`;
+        }
+
+        i = obj.attr.style.indexOf("text-decoration:");
+        if (i !== -1) {
+            underline = `${obj.attr.style[i + 1].replace(/;/g, "")}`;
+        }
+    }
```

Comment:
So `obj.attr.style` is something like `"background: blue; color: pink"`. `i` is 0 for background, so `obj.attr.style[i + 1].replace(/;/g, "")` results in `a`, the second character of the word "background".

How does this work?

Reply:
Sorry it took so long for me to reply, I didn't see a notification for your comment. But now I'm here and happy to explain!

So, the whole [i] [i+1] bit is a workaround stemming from the fact that html2json gives us the style attribute of the tag as an array instead of a key-value pair. So the style name always precedes the style value in the array (ex.: `["background", "blue"]`. In the code, to get the value of the background, you need to first find the index (`i`) of `"background"` in the array, and the value (`"blue"`) will be the element immediately following that (`i + 1`).

Given a string like `"<span style='color:#5b5b5b;'>[9593554]</span> <span style='#5b5b5b'>[</span>"`, html2json() will return an object with a structure similar to this:

```
{
    "node": "root",
    "child": [
        {
            "node": "element",
            "tag": "span",
            "attr": {
                "style": [
                    "color:",
                    "#5b5b5b;"
                ]
            },
            "child": [
                {
                    "node": "text",
                    "text": "[9593554]"
                }
            ]
        },
        {
            "node": "text",
            "text": " "
        },
        {
            "node": "element",
            "tag": "span",
            "attr": {
                "style": [
                    "color:",
                    "#5b5b5b;"
                ]
            },
            "child": [
                {
                    "node": "text",
                    "text": "["
                }
            ]
        }
    ]
}    
```
So, `obj.atrr.style` would equal `["color", "#5b5b5b"]`. 

If the span were `<span style="background: blue; color: pink;">foobar</span>`, then `obj.attr.style` = `["background", "blue", "color", "pink"]`.

So, the code snippet you linked from the file reads, in English:
If the `underline` style is specified in the tag's style attribute, then set the value of the variable `underline` to the value of the element immediately after (which will be 'underline'), and remove any semicolons from that string.

I'll add some examples to the README when I have a chance, hopefully later this evening or tomorrow.

Reply:
I've added a quick description to the README for the plugin.